dev-ruby/vagrant_cloud/vagrant_cloud-3.0.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_RECIPE_DOC="none"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Vagrant Cloud API Library"
+HOMEPAGE="https://github.com/hashicorp/vagrant_cloud"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+ruby_add_rdepend "
+	>=dev-ruby/excon-0.73
+	>=dev-ruby/log4r-1.1.10
+"
+ruby_add_bdepend ">=dev-ruby/rake-12.3
+	test? (
+		>=dev-ruby/webmock-3.0
+	)
+"
+
+all_ruby_prepare() {
+	# # loosen dependencies
+	sed -e 'require_relative' \
```

Comment:
This will break gemspec generation because the version is used later on. You can change require_relative to require ./path, there are several examples in dev-ruby already.

Reply:
Ok I upgrade this with 
```
sed -e 's:require_relative ":require "./:'
```